web/src/api.ts

```diff
@@ -225,6 +225,18 @@ export const payForCommit = async (
   return res.json();
 };
 
+/**
+ * Completes the commit with the specified commitId.
+ * @param commitId Id of the commit to complete
+ */
+export const completeCommit = async (commitId: number) => {
+  const res = await fetch(
```

Comment:
does this have auth check?

Reply:
@ifhuang I haven't added that yet. Will add in a later PR if there is enough time.